components/brave_news/browser/publishers_controller.h

```diff
@@ -71,8 +76,12 @@ class PublishersController {
 
   std::unique_ptr<base::OneShotEvent> on_current_update_complete_;
   base::ObserverList<Observer> observers_;
+
   std::string default_locale_;
-  Publishers publishers_;
+  // Threadsafe data for feed builder
+  scoped_refptr<base::RefCountedData<Publishers>> publishers_ =
```

Comment:
JFIY: if `publishers_` content could be modified on UI thread, it doesn't help to prevent a race.
`RefCountedData` provides thread-safety only for counter, not just for all wrapped data

Reply:
a relatively safe way check that it could be modified is to add `const`:
`base::RefCountedData<const Publishers>>`

Reply:
Good point. I totally agree.